bin/release-helper.sh

```diff
@@ -158,8 +158,8 @@ function cmd-set-dep-ver() {
     dep=$1
     ver=$2
 
-    grep -Eh "^(\s*\")${dep}(\[[a-zA-Z0-9]+\])?(>|=|<)(.*\",)" ${DEPENDENCY_FILE} || { echo "dependency ${dep} not found in ${DEPENDENCY_FILE}"; return 1; }
-    sed -i -r "s/^(\s*\")(${dep})(\[[a-zA-Z0-9,]+\])?(>|=|<)(.*\",)/\1\2\3${ver}\",/g" ${DEPENDENCY_FILE}
+    grep -Eh "^(\s*\")${dep}(\[[a-zA-Z0-9,\-]+\])?(>|=|<)(.*\",)" ${DEPENDENCY_FILE} || { echo "dependency ${dep} not found in ${DEPENDENCY_FILE}"; return 1; }
```

Comment:
in addition to alphanumeric characters and `-`, also `_` and `.` are admissible in dependency names afaik

Reply:
Thanks! I'll keep it for now though, I don't want to sneak too much in this change here for now... The regex is already complex enough 😄